test/metabase/models/collection_test.clj

```diff
@@ -328,38 +321,41 @@
                  :model/Collection {c2 :id} {:archived true :archive_operation_id "1234"}
                  :model/Collection {c3 :id} {:archived true :archive_operation_id "1234"}
                  :model/Collection {c4 :id} {:archived true :archive_operation_id "5678"}]
-    (let [visible-collection-ids (fn [c] (set/intersection (collection/visible-collection-ids c)
-                                                           #{c1 c2 c3 c4 (collection/trash-collection-id) "root"}))]
-      (with-current-user-perms-for-collections! [c1 c2 c3 c4]
+    (letfn [(visible-collection-ids [config]
```

Comment:
This test is broken, see https://metaboat.slack.com/archives/CKZEMT1MJ/p1725585478848519 -- while trying to debug it I made the output a little easier to read (symbols instead of random IDs) so it's easier to see what's wrong.

Reply:
CC @johnswanson, idk if you want to incorporate tweaks for easier debugging whenever you work on this test in the future